packages/tabs/src/Tabs.ts

```diff
@@ -161,7 +160,7 @@ export class Tabs extends SizedMixin(Focusable, { noDefaultSize: true }) {
     private slotEl!: HTMLSlotElement;
 
     @query('#list')
-    private tabList!: HTMLDivElement;
+    protected tabList!: HTMLDivElement;
```

Comment:
Once we make this protected, if we want to change these in the future, it becomes a breaking change for subclasses.